nixos/modules/services/logging/logstash.nix

```diff
@@ -88,7 +88,7 @@ in
         example = ''
           # Read from journal
           pipe {
-            command => "''${pkgs.systemd}/bin/journalctl -f -o json"
+            command => "${pkgs.systemd}/bin/journalctl -f -o json"
```

Comment:
The `''` tells the parser that the subsequent `${...}` is not an anti-quote (an expression whose value should be interpolated) but rather to treat it as a literal. See https://nixos.org/nix/manual/#ch-expression-language.